### PR TITLE
Improve dependency management

### PR DESCRIPTION
**Pull request by sandersky**
# PATCH

### CODE REVIEW

File: blueprints/ember-frost-core/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      'ember-frost-button',
+      'ember-frost-checkbox',
+      'ember-frost-css-core',
+      'ember-frost-icons',
+      'ember-frost-link',
+      'ember-frost-loading',
+      'ember-frost-password',
+      'ember-frost-select',
+      'ember-frost-text',
+      'ember-frost-textare',
```
Comment (sglanzer-fire) [inline blueprints/ember-frost-core/index.js]: Missing an 'a' at the end of 'area'

File: blueprints/ember-frost-core/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      .then(() => {
+        return this.addBowerPackagesToProject([
+          {name: 'lodash', target: '^4.10.0'}
+        ])
+      })
+      .then(() => {
+        return this.addAddonsToProject({
+          packages: [
+            {name: 'ember-computed-decorators', target: '>=0.2.2 <2.0.0'},
+            {name: 'ember-lodash', target: '>=0.0.6 <2.0.0'},
```
Comment (sglanzer-fire) [inline blueprints/ember-frost-core/index.js:29]: Does this handle adding lodash in bower/npm?  If so, can we remove the addBower section above?

File: package.json [hunk 3]
```diff
@@ -113,12 +112,14 @@
...
     "ember-cli-babel": "^5.1.5",
     "ember-cli-sass": "^5.2.0",
-    "ember-computed-decorators": "^0.2.2",
-    "ember-cli-htmlbars": "^1.0.1",
-    "ember-one-way-controls": "0.5.3",
-    "liquid-fire": "^0.22.0"
+    "ember-cli-htmlbars": "^1.0.1"
   },
   "ember-addon": {
+    "after": [
```
Comment (sglanzer-fire) [inline package.json:118]: Do we need the `after`?

**Comment by sglanzer-fire** [comment]
Approved - just need to fix the 'textarea' typo

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/47/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

File: blueprints/ember-frost-core/index.js (hunk 2 above)
Comment (sandersky) [inline blueprints/ember-frost-core/index.js:29]: No because `ember-lodash` doesn't have blueprints.

**Comment by sglanzer-fire** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/47/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

File: blueprints/ember-frost-core/index.js (hunk 2 above)
Comment (sglanzer-fire) [inline blueprints/ember-frost-core/index.js:29]: Lame - thanks for covering that 👍

**Pull request merged**